data_structures/linked_list/stack_using_linked_lists.c

```diff
@@ -36,40 +36,40 @@ int main()
     }
 }
 
-void push(struct node *p)
+void push(struct node *p) // push function will add a new node at the head of the list, time complexity O(1).
```

Comment:
For how to document functions, please see the [repository guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md).